src/main/webapp/i18n/en/exam.json

```diff
@@ -544,6 +544,7 @@
             "enable": "Enable",
             "disable": "Disable",
             "enabled": "Enabled",
+            "globallyDisabled": "Global disabled",
```

Comment:
```suggestion
            "globallyDisabled": "Globally disabled",
```

Reply:
I wasn't sure about that, but since you also suggested it we can use it :D thanks!